src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +19,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(
+                () -> new RuntimeException("Can't get movie by your id: " + id));
```

Comment:
It's better to inform user that `movie with by id not found`. Message with that construction is better for user because it explains user that we can find a movie, but the movie with that id does not exist.